charts/prometheus/values.yaml

```
@@ -65,7 +65,9 @@ configmapReload:
       digest: ""
       pullPolicy: IfNotPresent
 
-    # containerPort: 9533
+    ## config-reloader's container port and port name for probes and metrics
+    containerPort: 8080
+    containerPortName: metrics 
```

Please, remove the trailing space:
```suggestion
    containerPortName: metrics
```